packs/travisci/actions/hook.yaml

```diff
@@ -0,0 +1,15 @@
+name: hook
```

Comment:
Ok, I have noted the comments, I'll improve these and then push up the stream. Thanks